04-django/01-python-intro/min.py

```diff
@@ -0,0 +1,11 @@
+def min(array): 
+    try:
+        min_number = array[0]   
+        for x in array:
+            if x <= min_number:min_number = x      
+            else: min_number = min_number
+        return min_number
+    except IndexError as e:
+        print('array is empty')
```

Comment:
The good practice is to not print anything in case of errors.

I would either:

- Return a value as sensible as possible, like `0`,
- Raise a precise error if the array is empty: for example `raise Error("The array is empty.")`.

Reply:
So don't need to use try except ? 
How to close this request change once I fix it?